src/test/java/core/basesyntax/service/RegistrationServiceTest.java

```diff
@@ -0,0 +1,161 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.fail;
+
+import core.basesyntax.exception.AuthenticationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceTest {
+    private RegistrationServiceImpl registrationService;
+
+    @BeforeEach
+    void setUp() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void justAddingFitUsers_Ok() {
+        try {
+            registrationService.register(new User("vlad222r", "afsgggwa", 25));
+            registrationService.register(new User("bob222", "fjjsafj", 18));
+            registrationService.register(new User("alice222", "viadkc", 65));
+        } catch (AuthenticationException e) {
+            fail("Can't add user");
+        }
+    }
+
+    @Test
+    void checkNullUser_notOk() {
+        try {
+            registrationService.register(null);
+        } catch (AuthenticationException e) {
+            return;
+        }
+        fail("User is not made");
```

Comment:
Please use assertThrows() method in tests instead of try/catch to check if proper exception occured.
Same issue for all other negative tests